samigo/samigo-app/src/webapp/jsf/section-activity/sectionActivityHeadings.jsp

```diff
@@ -8,9 +8,9 @@
        <h:outputText value="#{generalMessages.assessment}" />
        <f:actionListener type="org.sakaiproject.tool.assessment.ui.listener.author.AuthorActionListener" />
     </h:commandLink>
-<f:verbatim></span></li>
+<f:verbatim rendered="#{authorization.adminTemplate and template.showAssessmentTypes}"></span></li>
```

Comment:
So this change above controls the rendering of an end span tag only ...... This doesn't make any sense to me.